### ISSUE TITLE
DROP TABLE syntax

### ISSUE THREAD

**Issue by walter-weinmann** [K2InformaticsGmbH/sqlparse#59]
According to the Oracle documentation it is `DROP TABLE ...` not `DROP tbl_type TABLE ...`.

**Comment by walter-weinmann** [comment]
The current definition

```SQL
drop_table_def -> DROP tbl_type TABLE opt_exists table_list opt_restrict_cascade                : {'drop table', {'tables', '$5'}, '$4', '$6', '$2'}.

drop_table_def ::= 'DROP' ( tbl_type )? 'TABLE' ( 'IF' 'EXISTS' )? ( table ( ',' table )* ) ( 'RESTRICT' | 'CASCADE' )?
```

will be changed to

```SQL
drop_table_def -> DROP      TABLE opt_exists table_list opt_restrict_cascade                    : {'drop table', {'tables', '$4'}, '$3', '$5', []}.
drop_table_def -> DROP NAME TABLE opt_exists table_list opt_restrict_cascade                    : {'drop table', {'tables', '$5'}, '$4', '$6', unwrap_bin('$2')}.

drop_table_def ::= 'DROP' ( NAME )? 'TABLE' ( 'IF' 'EXISTS' )? ( table ( ',' table )* ) ( 'RESTRICT' | 'CASCADE' )?
```

**Comment by walter-weinmann** [comment]
Solved in version 3.0.0.
